micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/CitizenSideBar.js

```
@@ -48,117 +56,153 @@ const Profile = ({ info, stateName, t }) => {
+  const stringReplaceAll = (str = "", searcher = "", replaceWith = "") => {
+    if (searcher == "") return str;
+    while (str?.includes(searcher)) {
+      str = str?.replace(searcher, replaceWith);
+    }
+    return str;
+  };
+
+  useEffect(() => {
+    const userloggedValues = Digit.SessionStorage.get("citizen.userRequestObject");
+    let teantsArray = [],
+      filteredArray = [];
```

Biome

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)